secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  DB_Host: c3FsMDE=
+  DB_Password: cGFzc3dvcmQxMjM=
+  DB_User: cm9vdA==
```

Comment:
I am assuming that you're aware that it's not a good practice to store secrets in git.
It's ok for this exercise but in real use cases we need to use more sophisticated approaches